frontend/src/lib/components/InsightCard/InsightDetails.tsx

```diff
@@ -123,23 +123,31 @@ function SeriesDisplay({
 }
 
 function PathsSummary({ filters }: { filters: Partial<FilterType> }): JSX.Element {
-    const humanEventTypes: string[] = []
+    // Sync with summarizePaths in utils
```

Comment:
why not extract this block as a function instead of syncing it?

Reply:
I was thinking about it, but I came to the conclusion that this'd be premature, because only the `if` part is exactly common between them. The issue is that overall they return very different things (`string` vs `JSX.Element`).

Reply:
Cool, easy enough to do in future if they stay similar and we sync them a lot 👍